lib/api.proto

```diff
@@ -44,9 +44,10 @@ message SyncRecord {
     Site site = 1;
     bool isFolder = 2;
     bytes parentFolderObjectId = 3;
-    bytes previousObjectId = 4;
-    bytes nextObjectId = 5;
-    repeated string fields = 6;
+    bytes previousObjectId = 4; // unused
+    bytes nextObjectId = 5; // unused
+    repeated string fields = 6; // unused
```

Comment:
hm i couldn't find anywhere it was used in browser-laptop but i'll remove the comment for now